packages/flame_mini_sprite/example/lib/main.dart

```diff
@@ -9,15 +9,15 @@ void main() {
 }
 
 const raw =
-    '16,16;7,0;2,1;13,0;1,1;2,0;1,1;12,0;1,1;2,0;1,1;12,0;1,1;2,0;1,1;12,0;1,1;2,0;1,1;12,0;1,1;2,0;1,1;11,0;1,1;1,0;2,1;1,0;1,1;10,0;1,1;1,0;2,1;1,0;1,1;10,0;1,1;1,0;2,1;1,0;1,1;9,0;1,1;1,0;1,1;2,0;1,1;1,0;1,1;7,0;1,1;2,0;1,1;2,0;1,1;2,0;1,1;5,0;1,1;2,0;2,1;2,0;2,1;2,0;1,1;3,0;1,1;2,0;1,1;2,0;2,1;2,0;1,1;2,0;1,1;1,0;1,1;2,0;2,1;1,0;1,1;2,0;1,1;1,0;2,1;2,0;2,1;2,0;10,1;2,0;1,1;1,0;2,1;10,0;2,1;1,0';
+    '16,16;20,-1;1,0;7,-1;1,0;8,-1;7,0;8,-1;1,0;7,-1;1,0;7,-1;1,0;7,-1;1,0;7,-1;1,0;1,-1;2,0;1,-1;2,0;1,-1;1,0;6,-1;1,0;9,-1;1,0;5,-1;1,0;2,-1;1,0;3,-1;1,0;2,-1;1,0;5,-1;1,0;2,-1;1,0;3,-1;1,0;2,-1;1,0;5,-1;1,0;2,-1;1,0;3,-1;1,0;2,-1;1,0;5,-1;1,0;2,-1;1,0;3,-1;1,0;2,-1;1,0;5,-1;1,0;9,-1;1,0;6,-1;9,0;51,-1';
 
 class MyGame extends FlameGame {
   @override
   Future<void> onLoad() async {
     final miniSprite = MiniSprite.fromDataString(raw);
     final sprite = await miniSprite.toSprite(
       pixelSize: 4,
-      color: Colors.white,
+      palette: [const Color(0xFFFFFFFF)],
```

Comment:
I wonder if it would maybe be better to have this as a map, then the colors could have keys instead of numbers in the format? But maybe it doesn't really give anything to the dev?

Reply:
Since this is something that the user wouldn't touch normally, I would prefer to keep it as number to keep the generated string small.

On a follow up PR I intend to create a `MiniPalette` class, which can abstract some stuff for it and maybe even have keys for the colors.